### DISCUSSION TITLE
changelog: Added comments to blog

### DISCUSSION THREAD

**Discussion by giscus** [jonathanyeong/personal-website#34] [Blog Post Discussions]
# changelog: Added comments to blog

Added Giscus comments to my blog

https://jonathanyeong.com/changelog-added-comments-to-blog/

**Comment by jonathanyeong** [comment]
Test comment

**Reply by jonathanyeong** [reply]
Test Reply

**Comment by jonathanyeong** [comment]
Testing gif
![grogu dancing](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExeHkyd3YxcmllcGVsbXI4aDlib3dqb2NxeGtteWdvbzgzMTQ2cDRnNCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/AcfTF7tyikWyroP0x7/giphy.gif)
